tuf-spec.md

```diff
@@ -173,9 +173,10 @@ repo](https://github.com/theupdateframework/specification/issues).
       of the problem.
 
       + **Malicious mirrors preventing updates.**  A repository mirror cannot
-      prevent updates from good mirrors. This includes mirrors that have been
-      taken down due to a DoS attack, malware, or pressure on administrators
-      (known as a permanent takedown threat).
+      prevent updates from good mirrors (such as by sharing malicious files).
+      Malicious mirrors include mirrors that have been taken down due to a
+      DoS attack, malware, or pressure on administrators (known as a permanent
+      takedown threat).
```

Comment:
Sorry, but I don't think you mean "taken down" for DoS attacks _and_ malware or pressure on administrators?